### PR TITLE
Implement the functionality to display the modal "Adjuntar"

### PR DESCRIPTION
**Pull request by Andresbl123**
Implement the functionality to display the modal on the attach `Button` on the `Application` page, this PR does not include the functionality of the modal to add or attach a new file. 

image

### CODE REVIEW

**Review by Angelbt12 — COMMENTED** [review]
good job

File: src/components/modals/Listmodal/index.tsx [hunk 1]
```diff
@@ -11,16 +11,26 @@ import {
...
 
 export interface IListmodalProps {
   title: string;
   portalId?: string;
   content?: JSX.Element | JSX.Element[];
+  optionButtons?: IOptionButtons;
   handleClose: () => void;
 }
 
 export const Listmodal = (props: IListmodalProps) => {
```
Comment (carin91) [inline src/components/modals/Listmodal/index.tsx:32]: Adjust the modal to align centers between the Text and the X icon
image

File: src/components/modals/Listmodal/index.tsx [hunk 2]
```diff
@@ -11,16 +11,26 @@ import {
...
-  const { title, portalId, content, handleClose } = props;
```
Comment (carin91) [inline src/components/modals/Listmodal/index.tsx:23]: i'm not sure about the expected behavior on "Cerrar", it should not be clickable?
image

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/modals/Listmodal/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/modals/Listmodal/index.tsx:32]: resolve

File: src/components/modals/Listmodal/index.tsx (hunk 2 above)
Comment (Andresbl123) [reply src/components/modals/Listmodal/index.tsx:23]: tener en cuenta que el componente `<Stack />` no tiene ejecuta el evento `onClick`.

Resolve

**Review by carin91 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**